### ISSUE TITLE
<html> in GitHub Star button violates WCAG 3.1.1 (upstream issue)

### ISSUE THREAD

**Issue by auchenberg** [microsoft/vscode-docs#2617]
html-has-lang: <html> element must have a lang attribute (1) WCAG 3.1.1

Url: https://code.visualstudio.com/docs/editor/versioncontrol

Links:
- https://www.w3.org/WAI/WCAG21/Understanding/language-of-page.html
- https://dequeuniversity.com/rules/axe/3.2/html-has-lang?application=msftAI

Code path:
```
<html>
```

**Comment by Nikhitha-13** [comment]
The same issue is observed through out the application in the following URLs:
1.code.visualstudio.com/download
2. code.visualstudio.com/docs/supporting/faq
3. code.visualstudio.com/docs/setup/Linux
4. code.visualstudio.com/docs/python/python-tutorial
5. code.visualstudio.com/docs/editor/whyvscode
6. code.visualstudio.com/docs/getstarted/tips-and-tricks
7. code.visualstudio.com/updates/v1_29
8. code.visualstudio.com/docs/editor/debugging
9. code.visualstudio.com/docs/editor/integrated-terminal
10. code.visualstudio.com/docs/editor/versioncontrol

**Comment by Nikhitha-13** [comment]
![Bug_#2617_issue](https://user-images.githubusercontent.com/50437925/57450049-0d3a9e00-727b-11e9-9d5a-056295997e18.png)
![Bug_#2617_issue](https://user-images.githubusercontent.com/50437925/57450068-1d527d80-727b-11e9-9118-32e524b5d9d7.png)
![Bug_#2617_issue(html element must have lang attribute)](https://user-images.githubusercontent.com/50437925/57450109-33f8d480-727b-11e9-92aa-72511aee5371.png)
![Bug_#2617_issue](https://user-images.githubusercontent.com/50437925/57450127-4c68ef00-727b-11e9-8b1b-0eb5016e064f.png)
![Bug_#2617_issue](https://user-images.githubusercontent.com/50437925/57450152-60acec00-727b-11e9-8672-a96ef7969785.png)
![Bug_#2617_issue](https://user-images.githubusercontent.com/50437925/57450182-76221600-727b-11e9-9b34-e9138de52032.png)
![Bug_#2617_issue](https://user-images.githubusercontent.com/50437925/57450218-8934e600-727b-11e9-9efc-8302bc5f7b1a.png)
![Bug_#2617_issue](https://user-images.githubusercontent.com/50437925/57450243-994cc580-727b-11e9-82f7-e0b68b5fc371.png)
![Bug_#2617_issue](https://user-images.githubusercontent.com/50437925/57450271-aec1ef80-727b-11e9-8e5c-240866d0f5b4.png)
![Bug_32617_issue](https://user-images.githubusercontent.com/50437925/57450321-c7320a00-727b-11e9-8846-83aaaae9cba1.png)

**Comment by auchenberg** [comment]
Upstream issue: https://github.com/mdo/github-buttons/issues/130

**Comment by auchenberg** [comment]
Closing as this is upstream issue caused by an external component. Once the upstream issue is fixed, this will be resolved.

**Comment by Nikhitha-13** [comment]
@auchenberg can you please reopen the bug as it is not yet fixed and close the current bug once the mentioned upstream issue gets closed.

**Comment by Nikhitha-13** [comment]
Yes can not see this issue now after running keros. So the issue is fixed. Please refer the below attachment.
![issue_fixed](https://user-images.githubusercontent.com/50437925/61031493-43acaa80-a3dd-11e9-94d0-8e36568fce81.png)

**Comment by fsteffi** [comment]
GitHubTags:#CAITestPass_June’19;
